ios/RNSScreenStack.mm

```diff
@@ -923,6 +923,7 @@ - (void)navigationController:(UINavigationController *)navigationController
 {
   [self emitOnFinishTransitioningEvent];
   [RNSScreenWindowTraits updateWindowTraits];
+  [_controller.view setNeedsLayout];
```

Comment:
One single note, let's leave a comment above this line why do we trigger layout here (e.g. link to this PR).